src/main/java/com/revature/pojos/Subscription.java

```diff
@@ -1,44 +1,66 @@
 package com.revature.pojos;
 
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="SUBSCRIPTION")
```

Comment:
table name is wrong, should be "SUBSCRIPTIONS"

Reply:
okk, missed that